app/gitcoinbot/actions.py

```diff
@@ -20,69 +20,161 @@
 import datetime
 import json
 import re
+from functools import wraps
 
 from django.conf import settings
 
 import jwt
 import requests
 import rollbar
+from dashboard.models import Bounty
+from dashboard.tokens import tokens
+from gitcoinbot.models import GitcoinBotResponses
 from github.utils import post_issue_comment_reaction
 
+MIN_AMOUNT = 0
+CURRENCIES = set(map(lambda currency: currency["name"], tokens))
+
+
+class Bound:
+    """Validate every bound before call the annotated function"""
```

Comment:
Can you add a trailing `.` to this docstring and add a new line between the docstring and `def __init__`